pyrasterframes/src/main/python/README.md

```diff
@@ -1,39 +1,49 @@
 # PyRasterFrames
 
-PyRasterFrames is the Python API for Spark RasterFrames.
+PyRasterFrames is a library for distributed processing of geospatial raster data with Spark.
 
 
 ## Prerequisites
 
-1. [`sbt`](https://www.scala-sbt.org/)
+1. [`pip`](https://pip.pypa.io/en/stable/installing/)
 2. ['pyspark`](https://pypi.org/project/pyspark/) > 2.3.2 
 
-RasterFrames is primarily implmented in Scala, and as such uses the Scala build tool [`sbt`](https://www.scala-sbt.org/).
-All `sbt` commands referenced below must be run from the root source directory, i.e. the parent of the `pyrasterframes` 
-directory, including Python-related build steps. 
-
-As a tip, know that `sbt` is much faster if run in "interactive" mode, where you launch `sbt` with no arguments, 
-and subsequent commands are invoked via an interactive shell. But for context clarity, we'll prefix each command 
-example below with `sbt`. 
-
 ## Quickstart
 
-The quickest way to run a `pyspark` shell with the latest RasterFrames enabled is to run:
+The quickest way to get started is to `pip` install the pyrasterframes package.
 
 ```bash
-sbt pySparkCmd
+pip install pyrasterframes
+```
+
+You can then access a [`pyspark SparkSession`]() using the [`local[*]` master](https://spark.apache.org/docs/latest/submitting-applications.html#master-urls) as follows.
+
+```python
+import pyrasterframes
+spark = pyrasterframes.get_spark_session()
+```
+
+Then you can read a raster and do some simple processing on it.
+
+```python
+from pyrasterframes.rasterfunctions import *
+from pyspark.sql.functions import lit
+# Read a Landsat 8 L1TP PDS scene
+df = spark.read.rastersource('https://landsat-pds.s3.amazonaws.com/c1/L8/038/037/LC08_L1TP_038037_20190322_20190403_01_T1/LC08_L1TP_038037_20190322_20190403_01_T1_B4.TIF')
+# Add 3 to every cell, show some rows of the dataframe
+df.select(rf_local_add(df.tile, lit(3))).show(6, False)
 ```
 
-This will: 
+## Development
```

Comment:
Would like to move these things out of this README and into a python development chapter in the docs. We can include that by reference.

```suggestion
## Development

Community contributions are always welcome. To get started, please review our [contribution guidelines](https://github.com/locationtech/rasterframes/blob/develop/CONTRIBUTING.md), [code of conduct](https://github.com/locationtech/rasterframes/blob/develop/CODE_OF_CONDUCT.md), and [developer's guide]().  Reach out to us on [gitter](https://gitter.im/locationtech/geomesa) so the community can help you get started!
```

Reply:
Need to replace `geomesa`...

Reply:
haha i copied my gitter tab address right in here.

Reply:
For now going to move developer centric docs to the pyrasterframes/README.md